scout/advanced-image-analysis.md

```diff
@@ -77,28 +86,25 @@ where Advanced image analysis is active:
 
 To view the vulnerability report on Docker Hub:
 
-1. Go to Docker Hub and open the repository page. A vulnerability summary is
-   available in the **Tags** section of the page.
+1. Go to Docker Hub and open the repository page. The **Tags** section
+   displays a vulnerability summary.
 
-   It may take a few minutes for the vulnerability report to appear in your
-   repository. If your vulnerability summary doesn't display yet, wait a moment
+   It may take a few minutes for the vulnerability report to appear. If your vulnerability summary doesn't display, wait a moment
    and then refresh the page.
 
 2. Click on the tag in the table. This opens the details page for the tag.
 
 3. Select the **Vulnerabilities** tab on the right side of the page.
 
-   This tab displays a deep-dive view of your vulnerability exposure. If your
-   image contains packages known to be affected by vulnerabilities, those
-   packages show up here.
+   This tab displays a deep-dive view of the image's packages and any known vulnerabilities.
 
    For more information about how to interpret the vulnerability report, see
-   [Image details view](./image-details-view.md)
+   [Image details view](./image-details-view.md).
 
 Expanding any of the packages in the list shows you more information about the
-vulnerabilities that affect a given package. Clicking on the hyperlink of a
-vulnerability opens it in Docker's image vulnerability database, which provides
-even more information on the vulnerability and what images it affects.
+vulnerabilities that affect a given package. Expanding the vulnerability shows a summary of it's details and
+clicking on the vulnerability name opens Docker's image vulnerability database, which provides
```

Comment:
```suggestion
selecting the vulnerability name opens Docker's image vulnerability database, which provides
```
We try to avoid using 'click' https://docs.docker.com/contribute/style/recommended-words/

Reply:
@aevesdocker Yes indeed, I saw that in VSCode too. For some reason when I read it, it didn't sound right in this context, but now it does, so 🤷‍♂️